modules/core/src/main/java/org/apache/ignite/cache/query/index/AbstractIndex.java

```diff
@@ -15,25 +15,31 @@
  * limitations under the License.
  */
 
-package org.apache.ignite.internal.processors.query.h2.database.inlinecolumn;
```

Comment:
Actually I don't know why GitHub creates the link StringIgnoreCaseInlineIndexColumn -> AbstractIndex. `git log` for AbstractIndex is the only commit. I tried to git mv, git rm, create file from scratch again. But nothing helps and GitHub still show this again. 

The same time Intellij IDEA shows history of this file differently, it saw it as GridQueryRowCacheCleaner -> AbstractIndex.

```
% git log modules/core/src/main/java/org/apache/ignite/cache/query/index/AbstractIndex.java
commit cee7855760b3fa1867a9e44e52be4ef786f108ad
Author: Maksim Timonin <timonin.maxim@gmail.com>
Date:   Thu Dec 10 01:25:18 2020 +0300

    Rebuild indexes, fix tests
```

If somebody have any idea how to fix this, please share this.